src/main/java/org/tron/core/db/PendingManager.java

```diff
@@ -29,7 +29,7 @@ public void close() {
         dbManager.getRepushTransactions().put(tx);
       } catch (InterruptedException e) {
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Either re-interrupt this method or rethrow the "InterruptedException". ([squid:S2142](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AS2142))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1324&issues=AWVbUexsZiUEgfxFv6nA&open=AWVbUexsZiUEgfxFv6nA)

[ISSUE_KEY]: AWVbUexsZiUEgfxFv6nA